sql/core/src/test/scala/org/apache/spark/sql/UDFSuite.scala

```diff
@@ -840,7 +840,7 @@ class UDFSuite extends QueryTest with SharedSparkSession {
     Row(ArrayBuffer(100)))
 
      val myUdf2 = udf((a: immutable.ArraySeq[Int]) =>
-      immutable.ArraySeq.unsafeWrapArray[Int](a.appended(5).appended(6).toArray))
+      immutable.ArraySeq.unsafeWrapArray[Int]((a :+ 5 :+ 6).toArray))
```

Comment:
Only fix the comment related issues of another PR: https://github.com/apache/spark/pull/44591

Reply:
I think this show be a follow up of https://github.com/apache/spark/pull/44591

Reply:
Could we move this change to a separate pr

Reply:
Done.